index.html

```diff
@@ -10,7 +10,7 @@
   <body>
     <div class="honeymoon-wrapper">

```

Comment:
bu div'in de dışında içinde ki büyük div'i pozisyonlamak için bir div olması gerekiyor

Reply:
Bu bahsettiğiniz divi daha önce açmıştım , onu silebilirsin gerek yok demiştiniz.. Sorun yok şimdi tekrardan açtım.. :)